packages/pharos-site/src/components/Sidenav.tsx

```diff
@@ -55,7 +55,7 @@ const Sidenav: FC = () => {
     };
 
     const content = (
-      <PharosSidenav mainContentId="skip-link" className={sidenav}>
+      <PharosSidenav mainContentId="skip-link" open={true} className={sidenav}>
```

Comment:
Just a note this will only partially fix this - we will be missing the expected mobile behavior. I think this is the right thing to do for now though, but will need a followup story probably too